dfdaemon/config/config.go

```diff
@@ -88,18 +88,14 @@ type Properties struct {
 	CertPem string `yaml:"certpem" json:"certpem"`
 	KeyPem  string `yaml:"keypem" json:"keypem"`
 
-	// dfget config
-	SuperNodes []string `yaml:"supernodes" json:"supernodes"`
-	DFRepo     string   `yaml:"localrepo" json:"localrepo"`
-	DFPath     string   `yaml:"dfpath" json:"dfpath"`
-	RateLimit  string   `yaml:"ratelimit" json:"ratelimit"`
-	URLFilter  string   `yaml:"urlfilter" json:"urlfilter"`
-	CallSystem string   `yaml:"callsystem" json:"callsystem"`
-	Notbs      bool     `yaml:"notbs" json:"notbs"`
-
 	Verbose bool `yaml:"verbose" json:"verbose"`
 
 	MaxProcs int `yaml:"maxprocs" json:"maxprocs"`
+
+	// dfget config
+	GlobalDfgetParams []string `yaml:"global_dfget_params" json:"global_dfget_params"`
```

Comment:
I don't think this should be a slice. Can we define a struct for this such as `DfgetParams`? Then we can reuse it in proxy. And we can do some validation using this struct.

Reply:
If that, we have to redefine a struct just like `dfget config`.

Reply:
make sense

Reply:
just use `dfgetflags`? And it should be assigned a default value.